### PR TITLE
Doc: Hardware setup

### PR DESCRIPTION
**Pull request by DanD1511**
Documentation for Arduino setup

### CODE REVIEW

File: content/en/docs/guides/conections_/index.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: "Zumo Shield v1.2 - Raspberry Pi 4 wire connection"
+linktitle: "Wire implementation"
```
Comment (jadarve) [inline content/en/docs/guides/conections_/index.md]: Not sure "implementation" is the right word here.

File: content/en/docs/guides/conections_/index.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+collapsible: false
+---
+
+
+## Wire connection
+
+{{< alert title="DataSheet" >}}
+You can check datasheet of Raspberry Pi 4 from [DataSheet](https://datasheets.raspberrypi.com/rpi4/raspberry-pi-4-datasheet.pdf).
+
+**You can check the following steps using DataSheet.**
```
Comment (jadarve) [inline content/en/docs/guides/conections_/index.md]: Not sure this line is needed.

File: content/en/docs/guides/conections_/index.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+date: 2023-07-20
+weight: 1
+collapsible: false
+---
+
+
+## Wire connection
+
+{{< alert title="DataSheet" >}}
+You can check datasheet of Raspberry Pi 4 from [DataSheet](https://datasheets.raspberrypi.com/rpi4/raspberry-pi-4-datasheet.pdf).
```
Comment (jadarve) [inline content/en/docs/guides/conections_/index.md]: This line could be improved.

File: content/en/docs/guides/setup/android_start/index.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+title: "Android start"
+linktitle: "Download APK"
+date: 2023-07-24
+weight: 5
+collapsible: false
+---
+
+You can download the APK following the next link: [APK](https://github.com/aldajo92/AndroidKt_JoyWebSocket/releases/download/1.0/JoyWebSocket.apk).
+
+After you intall the APK on your phone, just connect to the Raspberry Pi 4 ip.
```
Comment (jadarve) [inline content/en/docs/guides/setup/android_start/index.md]: How?

File: content/en/docs/guides/setup/android_start/index.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+---
+title: "Android start"
+linktitle: "Download APK"
```
Comment (jadarve) [inline content/en/docs/guides/setup/android_start/index.md]: `title` and `linktitle` are too different.

File: content/en/docs/guides/setup/android_start/index.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+date: 2023-07-24
+weight: 5
```
Comment (jadarve) [inline content/en/docs/guides/setup/android_start/index.md]: why 5?

File: content/en/docs/guides/setup/android_start/index.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+collapsible: false
+---
+
+You can download the APK following the next link: [APK](https://github.com/aldajo92/AndroidKt_JoyWebSocket/releases/download/1.0/JoyWebSocket.apk).
+
+After you intall the APK on your phone, just connect to the Raspberry Pi 4 ip.
+
+![](app.jpg)
+
+> **Note:** You can found the code of the apk in the next link [APK documentation](https://github.com/aldajo92/AndroidKt_JoyWebSocket).
```
Comment (jadarve) [inline content/en/docs/guides/setup/android_start/index.md]: This statement could be improved.

**Review by jadarve — COMMENTED** [review]
We could merge this PR once the suggested changes are included.

We probably need to go through another round of editions once we have this one and #4 merged.

File: content/en/docs/guides/conections_/index.md (hunk 1 above)
Comment (DanD1511) [reply content/en/docs/guides/conections_/index.md]: Solved

File: content/en/docs/guides/conections_/index.md (hunk 2 above)
Comment (DanD1511) [reply content/en/docs/guides/conections_/index.md]: Solved

File: content/en/docs/guides/conections_/index.md (hunk 3 above)
Comment (DanD1511) [reply content/en/docs/guides/conections_/index.md]: Solved

File: content/en/docs/guides/setup/android_start/index.md (hunk 4 above)
Comment (DanD1511) [reply content/en/docs/guides/setup/android_start/index.md]: Solved

File: content/en/docs/guides/setup/android_start/index.md (hunk 6 above)
Comment (DanD1511) [reply content/en/docs/guides/setup/android_start/index.md]: Solved

File: content/en/docs/guides/setup/android_start/index.md (hunk 7 above)
Comment (DanD1511) [reply content/en/docs/guides/setup/android_start/index.md]: Solved

File: content/en/docs/guides/conections_/index.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+linktitle: "Raspberry Pi 4 and Zumo Shield v1.2 connection" 
+date: 2023-07-20
+weight: 1
+collapsible: false
+---
+
+
+## Connection
+
+The next image shows the connection between the **Raspberry Pi 4** and **ZumoShield v1.2**
```
Comment (aldajo92) [inline content/en/docs/guides/conections_/index.md]: "The image below shows..." looks better.

File: content/en/docs/guides/setup/android_start/index.md [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+date: 2023-07-24
+weight: 1
+collapsible: false
+---
+
+{{< alert title="APK download" >}}
+Download the APK following the next link: [APK](https://github.com/aldajo92/AndroidKt_JoyWebSocket/releases/download/1.0/JoyWebSocket.apk).
+{{< /alert >}}
+
+After installing the APK on your phone, connect using the Raspberry Pi 4 IP address.
```
Comment (aldajo92) [inline content/en/docs/guides/setup/android_start/index.md]: you forgot to mention it is possible to connect to raspberry with the hostname also.

File: content/en/docs/guides/setup/android_start/index.md (hunk 5 above)
Comment (DanD1511) [reply content/en/docs/guides/setup/android_start/index.md]: solved

File: content/en/docs/guides/setup/android_start/index.md (hunk 9 above)
Comment (DanD1511) [reply content/en/docs/guides/setup/android_start/index.md]: solved

File: content/en/docs/guides/conections_/index.md (hunk 8 above)
Comment (DanD1511) [reply content/en/docs/guides/conections_/index.md]: solved

**Review by aldajo92 — APPROVED** [review]

**Pull request merged**